Public/Src/Engine/Scheduler/IPipScheduler.cs

```diff
@@ -21,7 +21,7 @@ public interface IPipScheduler
         /// Initialize runtime state, optionally apply a filter and schedule all ready pips.
         /// Do not start the actual execution.
         /// </summary>
-        bool InitForMaster([NotNull]LoggingContext loggingContext, RootFilter filter = null, SchedulerState state = null, IKextConnection kextConnection = null);
+        bool InitForMaster([NotNull]LoggingContext loggingContext, RootFilter filter = null, SchedulerState state = null, ISandboxConnection SandboxConnectionKext = null);

```

Comment:
please change to lowercase to avoid confusion, or maybe do a code base wide search and replace and chose something shorter?